### PR TITLE
Configure edit and register buttons for hosts and non-hosts

### PR DESCRIPTION
**Pull request by dash102**
image

Fixes issues #149 and #163

### CODE REVIEW

File: project/src/main/webapp/event-details.js [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
   const eventHost = await getEventHost();
+  configureRegisterAndEditButtons(eventHost);
   const loginStatus = await getLoginStatus();
   populateVolunteeringOpportunitiesUI(eventHost, loginStatus);
   showCreateOpportunityLink(eventHost, loginStatus);
   setSignupAction();
 };
 
+/** 
+ * Adds the edit button for the host and removes the register button
```
Comment (yicyao) [inline project/src/main/webapp/event-details.js:12]: If this removes the register button for the host, in what case would you get the alert for 'you cant register bc you are the host'? is it if the user manually navigates to the url?

File: project/src/main/webapp/event-details.js [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+ * @param {Object} eventHost email of the host of the event
+ * @param {Object} loginStatus status of current user
```
Comment (yicyao) [inline project/src/main/webapp/event-details.js:14]: seems to not exist

File: project/src/main/webapp/event-details.js (hunk 1 above)
Comment (dash102) [reply project/src/main/webapp/event-details.js:12]: They're 2 separate pages - the alert shows up when you click on register in the feed, and once you click into event details, it hides that register button

File: project/src/main/webapp/event-details.js (hunk 2 above)
Comment (dash102) [reply project/src/main/webapp/event-details.js:14]: Oops, didn't update that - fixed!

**Pull request merged**